src/components/ui/Skeleton/Skeleton.tsx

```
@@ -4,14 +4,30 @@
     // If loading is false, return the children
     if (!loading) return children;
 
     const rootClass = customClassSwitcher(customRootClass, COMPONENT_NAME);
-    return <span className={clsx(rootClass, className)} {...props} >
-        {children}
-    </span>;
+    return <div className={clsx(rootClass, className)} {...props} style={{
+    
+        ['--skeleton-height' as any]: height,
+        ['--skeleton-width' as any]: width,
+        ['--skeleton-radius' as any]: radius,
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)